package.json

```diff
@@ -23,40 +23,44 @@
     "prop-types": "^15.5.7"
   },
   "peerDependencies": {
-    "react": "15.x || 16.x"
+    "react": "15.x || 16.x || 17.x"
```

Comment:
15.x won't work with UNSAFE_ methods.

Reply:
I think min needs to be 16.3 now.

Reply:
You're right, I forgot that UNSAFEs were added only with 16.3.. Thanks for review, hopefully @clauderic can look into it, since a lot of people have to spend time forking the repo.